src/root-reducer.js

```diff
@@ -4,12 +4,13 @@ import {drinkReducer} from "./redux/reducers/drinkReducer";
 import {salatReducer} from "./redux/reducers/salatReducer";
 import {discountReducer} from "./redux/reducers/discountReducer";
 import basketReducer from "./redux/reducers/basketReducer";
+import alertReducer from "./redux/reducers/alertReducer";
```

Comment:
here you have import from (export default), above you have all the imports from (export)